### PR TITLE
Update buildpack extensions.

### PR DESCRIPTION
**Pull request by codynguyen1**
Need to be able to add .graphql file.

### CODE REVIEW

File: packages/cpse-template/src/intercept.js [hunk 1]
```diff
@@ -14,6 +14,6 @@ module.exports = targets => {
          *  Wee need to activated esModules and cssModules to allow build pack to load our extension
          * {@link https://magento.github.io/pwa-studio/pwa-buildpack/reference/configure-webpack/#special-flags}.
          */
-        flags[targets.name] = {esModules: true, cssModules: true};
+        flags[targets.name] = { esModules: true, cssModules: true, graphqlQueries: true };
```
Comment (larsroettig) [inline packages/cpse-template/src/intercept.js:17]: This flag leads to performance impact for all modules that don't want to induce own queries. 
I will write a FAQ page where I documented this issue and how to solve it. 

Thank you for your contribution.

**Comment by larsroettig** [comment]
@Frodigo what do you mean should I add this to default behavior ?

**Comment by Frodigo** [comment]
Maybe there is a way to prompt about this and write configuration based on answers?

Something like we have with package.json. A developer writes their name, name of the module, etc.

**Comment by larsroettig** [comment]
Not relay then we lost flexibility for custom templates.
I think I will document in FAQ Page and link this Pull Request thank you for using this open source tool.

**Pull request closed without merging**